pymagicc/core.py

```diff
@@ -1258,17 +1287,17 @@ def default_config(self):
 
         return self._default_config
 
-    def _check_tcr_ecs_total_RF(self, df_total_rf, tcr_time, ecs_time):
-        super()._check_tcr_ecs_total_RF(df_total_rf, tcr_time, ecs_time)
+    def _check_tcr_ecs_tcre_total_RF(self, df_total_rf, tcr_time, ecs_time):
```

Comment:
Why perform extra checks for the subclass. Is there a real world example which passes the Base classes implementation, but fails this check?

Reply:
Yes, running these checks with MAGICC7 doesn't work because it's CO2 RF isn't always logarithmic.